<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notebookClass</class>
 <widget class="QMainWindow" name="notebookClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>notebook</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 234, 234);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_btnLast">
          <property name="minimumSize">
           <size>
            <width>56</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>56</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>上一题</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnCtrl">
          <property name="minimumSize">
           <size>
            <width>56</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>56</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>显示题解</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnNext">
          <property name="minimumSize">
           <size>
            <width>56</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>56</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>下一题</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="m_edtQuestion">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="m_edtAnswer"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="m_treeQuestionBank">
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
